Chess/include/Board.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+#include <iostream>
+#include <vector>
+#include <string>
+//#include "Piece.h"
+
+using std::string;
+
+class Piece;
+class King;
+
+class Board
+{
+public:
+	Board();
+	~Board();
+	bool isEmpty(int, int);
+	Piece* getPiece(int, int);
+	bool isValidCoordinate(int, int) const;
+	int movePiece(string);
+
+private:
+	bool checkForCheck(bool);
+	std::vector<std::unique_ptr<Piece>> _pieces;
+	King* _whiteKing;
+	King* _blackKing;
+	bool _turn; // true is white's turn
```

Comment:
These variables are data members in the class, so their name should be according to the example: m_whiteKing